src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotifications.java

```diff
@@ -146,6 +148,15 @@ private static String getHash(@CheckForNull SCMRevision revision) {
         @Override
         public void onCheckout(Run<?, ?> build, SCM scm, FilePath workspace, TaskListener listener, File changelogFile,
                                SCMRevisionState pollingBaseline) throws Exception {
+
+            String delimiter = System.getProperty(WorkspaceList.class.getName(), "@");
+            if (workspace.getRemote().endsWith(delimiter + "script")) {
```

Comment:
This is a screaming hack... needs at least a comment alerting to the hackiness

Reply:
There is a comment below:
“this checkout is done to read the Jenkinsfile
that means there will be another checkout
 so no need to notify Bitbucket twice”

is there something else you would like to add?

this fix would temporarly while we get a fix to checkout only once for PRs with merge strategy. I think PR 117 and PR 123 might fix it. After that is complete you could remove this...

Reply:
Something like

```
// FIXME HACK this is just temporary pending JENKINS-...
```